cli-plugins/apps/create-workload.hbs.md

```diff
@@ -144,13 +179,13 @@ The intent of these bindings is to provide information from a service resource t
     - `pet-clinic` is the name of the workload to be updated.
     - `--service-ref` references the service using the format {service-ref-name}={apiVersion}:{kind}:{service-binding-name}.
 
-Check [services consumption documentation](../../getting-started/consume-services.md) to get more info on how to bind a service to a workload.
+Check [services consumption documentation](../../getting-started/consume-services.hbs.md) to get more info on how to bind a service to a workload.
 
 ## <a id="next-steps"></a> Next steps
 
-You can check workload details and status, add environment variables, export definitions, bind services and use flags with these [commands](command-reference.md). See more detailed explanation of flags usage for each command in [Command reference](command-reference.md).
+You can check workload details and status, add environment variables, export definitions or bind services.
 
-1. To check workload status and details, use `workload get` command and to get workload logs, use `workload tail` command. For more info about these, refer to [debug workload section](debug-workload.md).
+1. To check workload status and details, use `workload get` command and to get workload logs, use `workload tail` command. For more info about these, refer to [debug workload section](debug-workload.hbs.md).
```

Comment:
1. To check workload status and details, use `tanzu apps workload get`.</br>
To get workload logs, use `tanzu apps workload tail`.</br>
For more info about these, refer to [debug workload section](debug-workload.hbs.md).